app/Main.hs

```diff
@@ -37,10 +39,12 @@ toDigitsRev n = undefined
 -- and use DL.map..
 -- but remember it is all about writing Recursion at this point..
 doubleEveryOther :: [Integer] -> [Integer]
-doubleEveryOther list = undefined
+doubleEveryOther (x:y:xs) = x:(2 * y):doubleEveryOther xs
+doubleEveryOther x = x
 
 sumDigits :: [Integer] -> Integer
-sumDigits list = undefined
+sumDigits [x] = x
+sumDigits (x:xs) = x + sumDigits xs
 
 validate :: Integer -> Bool
```

Comment:
Also can you make another file `main_abhiroop.hs` as i would like to keep `main.hs` so that anyone can come and see the exercise.